README.md

```diff
@@ -13,6 +13,7 @@ Microverse Portfolio Project
 - Use images and backgrounds to enhance the look of the website.
 - Add second section (Work-About).
 - Add contact Form.
+- Add Desktop version with media queries.
 
 ## Built With
```

Comment:
- [ ] Good job so far 👏. I would suggest **you add two important sections.** The first one is the `Getting started`  section where you add the steps to clone and run your project locally. This is an interesting section for your fellow developers because It helps them to learn from your codes 😉. Another section is `Screenshot` section where you add the screenshot of your landing page. This gives a quick preview of what your project looks like 😎 👍